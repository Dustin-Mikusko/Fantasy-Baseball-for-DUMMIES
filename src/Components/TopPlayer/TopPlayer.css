.top-player-card {
  color: #041F42;
  border: 7px ridge #041F42;
  width: 65%;
  height: 160px;
  font-size: 1.4em;
  padding: 15px;
  margin: auto;
  margin-top: 40px;
  position: relative;
}

.top-player-name {
  text-align: center;
  border-bottom: 1px solid #041F42;
}

.stat {
  font-size: 1.2em;
}


