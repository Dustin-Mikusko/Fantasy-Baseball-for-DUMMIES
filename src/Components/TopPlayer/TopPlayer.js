import React from 'react';
import './TopPlayer.css'

export const TopPlayer = ({ stat, statName, team, name, id }) => {
  const statNameUpper = statName.toUpperCase();
  return (
    <>
      <h3>Name: {name}</h3>
      <p>Team: {team}</p>
      <p>{statNameUpper}: {stat}</p>
    </>
  )
}

export default TopPlayer;
