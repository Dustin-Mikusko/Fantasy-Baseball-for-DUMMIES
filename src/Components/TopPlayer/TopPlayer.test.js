import React from 'react';
import TopPlayer from './TopPlayer';
import { shallow } from 'enzyme';

describe('TopPlayer', () => {
  it('should match the snapshot with the correct data', () => {
    const mockPlayer = {
      stat: 'hr',
      team: 'Houston Astros',
      name: 'Jose Altuve',
      rank: '1'
    };
    const wrapper = shallow(<TopPlayer 
      statName='rbi'
      player={mockPlayer}
    />);

    expect(wrapper).toMatchSnapshot();
  })
})
