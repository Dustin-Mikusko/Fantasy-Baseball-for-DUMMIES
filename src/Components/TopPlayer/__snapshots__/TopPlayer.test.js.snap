// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopPlayer should match the snapshot with the correct data 1`] = `ShallowWrapper {}`;
